spec/requests/classified_listings_spec.rb

```diff
@@ -1,27 +1,27 @@
 require "rails_helper"
```

Comment:
shouldn't this file be renamed?

Reply:
The problem is we always had two specs here, `classified_listings_spec.rb` and `listings_spec.rb`. So simple renaming won't work. I wanted to figure out if it would be sensible to just merge them (probably is), but decided to tackle that a bit later.